<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKYY Codka</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Icons/SoftwareIcon.png</normaloff>:/Icons/SoftwareIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-color: rgb(61, 61, 61);
}

QToolButton{
	background-color: rgba(0, 0, 0, 20%);
	font: 20px 'EngraversGothic BT';
	color: White;
}

QLineEdit{
	font: 20px 'EngraversGothic BT';
}

QTextEdit{
	background: transparent;
	font: 15px 'EngraversGothic BT';
	color: White;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="buttonBuildRun">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="text">
         <string>Build and Run</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/Icons/Socialism.png</normaloff>:/Icons/Socialism.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="layoutEditor"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textMessages">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
        <property name="placeholderText">
         <string>Informações serão exibidas aqui...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
